securedrop-proxy/debian/securedrop-proxy.triggers

```diff
@@ -1,7 +1,7 @@
 # Register interest in Python interpreter changes (Python 2 for now); and
 # don't make the Python package dependent on the virtualenv package
 # processing (noawait)
-interest-noawait /usr/bin/python3.5
```

Comment:
ah so only the changes in the `securedrop-proxy/` directory will be needed here

Reply:
whoops went a little overboard with find-and-replace